ipaplatform/debian/services.py

```diff
@@ -35,6 +35,7 @@
 debian_system_units['rpcgssd'] = 'rpc-gssd.service'
 debian_system_units['rpcidmapd'] = 'nfs-idmapd.service'
 debian_system_units['smb'] = 'smbd.service'
+debian_system_units['sssd'] = 'sssd.service'
```

Comment:
@abbra asked me to add the service there. 

Did I understand you correct? If a service already is in `ipaplatform.base.services.wellknownservices` even without '.service' suffix ('sssd' in that case), there is no need to add the service to `ipaplatform.debian.services` and `ipaplatform.redhat.services` in case name of the service is the same for all systems, e.g. for redhat and debian. Right?

Reply:
That's correct. ``redhat_system_units`` and ``debian_system_units`` are overrides in case the systemd service has a different name than we expect.

Reply:
Thanks for the explanation